05LessonFive/Checkpoint1/index.html

```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Landscape Design</title>
+  <title>Berries</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
-
-</body>
+<main>
+  <section class="container clearfix">
+    <div class="image-holder"><img src="http://hd.freewallpaper-s.net/wp-content/uploads/mixed2/raspberries-tea-cup-top-2560x1440.jpg" width="500px"></div>
```

Comment:
try adjusting the browser width--do you see how the image's width is fixed, and the text box will overlap it? that's because your inline width declaration is overriding your external stylesheet, so no matter what, the raspberry image will be 500px wide. i'm going to record a walkthrough of this project tonight--should be posted by the end of the week!